src/physics/fermions.jl

```diff
@@ -34,7 +34,7 @@ end
 
 isfermionic(i::Index) = has_fermionic_subspaces(i)
 
-has_fermionic_subspaces(is::IndexSet) = false
+has_fermionic_subspaces(is::Union{IndexSet,NTuple{N,Index}}) where {N} = false
```

Comment:
`IndexSet` was deprecated a while ago, you could use `Indices` here:
https://github.com/ITensor/ITensors.jl/blob/3ac68593e13d383d9a50ccf386aee15a73dcbaea/src/indexset.jl#L24

Reply:
Good to know – changing it now thanks